setup.py

```diff
@@ -45,7 +45,7 @@
     "click>=7",
     "pathspec==0.9.0",
     "jsonpointer>=2.0",
-    "reana-commons[yadage,snakemake,cwl]>=0.9.4a1,<0.10.0",
+    "reana-commons[yadage,snakemake,cwl]>=0.9.4a4,<0.10.0",
```

Comment:
```suggestion
    "reana-commons[yadage,snakemake,cwl]>=0.9.4a3,<0.10.0",
```
After this CI should pass!